pyanaconda/ui/gui/hubs/__init__.py

```diff
@@ -60,6 +60,7 @@ class Hub(GUIObject, common.Hub):
     """
```

Comment:
Also, the loops are disabled/enabled when the hub is show, not when it is created, AFAICT.